### DISCUSSION TITLE
ERROR: Could not find a single window on screen

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [tauri-apps/tauri#5403] [Q&A]
I build my application according to tauri docs and use ubuntu 18. When I publish a new appimage into Appimage.org. I face `" ERROR: Could not find a single window on screen :-(" exit 1`
To solve this, according to the Appimage error config, the `terminal=true` in appimage *.desktop file. But I can't find any docs relevant to my error.

```
[Desktop Entry]
Categories=Education;
Comment=Convert nextjs static website to the cross-desktop app with tauri
Exec=static-blog-app
Icon=static-blog-app
Name=static-blog-app
Terminal = false   // How to change false  to Terminal=true
Type=Application
X-AppImage-Version=b965c99
```

Read the full detail error on https://github.com/AppImage/appimage.github.io/pull/3026.

**Comment by FabianLars** [comment]
We don't have a built-in way to modify the .desktop file (tracking issue: https://github.com/tauri-apps/tauri/issues/5176), but in theory you could unpack the appimage and re-pack it manually, _but_ setting Terminal to true sounds completely wrong to me for Tauri apps 🤔
